hedera-node/javax-inject/build.gradle.kts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2020-2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+plugins {
```

Comment:
We should not be repackaging third-party code without our legal team reviewing and approving. Additionally, why are we repackaging third-party code which is already available on Maven Central under the `javax.inject:javax.inject:1` vector?

Reply:
Hi @nathanklick, we didn't want to include this third-party code repackaging, however we couldn't figure out a way to require this library as a module dependency inside `hedera-evm-api`. That's why we decide to repackage the library with added custom module declaration. We can remove it, but could you suggest a way to add the library from Maven Central as a module dependency?

Reply:
Managed to fix this with `org.gradlex.extra-java-module-info`. Thanks, @nathanklick!